bears/__init__.py

```diff
@@ -20,3 +22,28 @@ def assert_supported_version():  # pragma: no cover
     if sys.version_info < (3, 4):
         print('coala supports only python 3.4 or later.')
         exit(4)
+
+
+def check_coala_version():
+    """
+    Check the installed coala and coala-bears version when the coalib loads
```

Comment:
Just "Check the installed coala and coala-bears version." so it fits onto one line (pep257), and ... it doesnt matter how the coala-bears package is loaded ... this will run.